examples/advanced/create-react-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "style-dictionary-create-react-app-sass",
+  "description": "",
+  "version": "1.0.0",
+  "dependencies": {
+    "node-sass": "^4.10.0",
+    "react": "^16.6.1",
+    "react-dom": "^16.6.1",
+    "react-scripts": "2.1.1",
+    "styled-components": "^4.1.2"
+  },
+  "devDependencies": {
+    "style-dictionary": "2.4.0"
+  },
+  "scripts": {
+    "build-dictionary": "node_modules/.bin/style-dictionary build --config ./style-dictionary/config.json",
```

Comment:
I think we can remove 'node_modules/.bin/' from this command

Reply:
done.